src/thinblock.cpp

```diff
@@ -1072,8 +1072,8 @@ std::string CThinBlockData::ToString()
     LOCK(cs_thinblockstats);
     double size = double(nOriginalSize() - nThinSize() - nTotalBloomFilterBytes());
     std::ostringstream ss;
-    ss << nBlocks() << " thin " << ((nBlocks() > 1) ? "blocks have" : "block has") << " saved " << formatInfoUnit(size)
-       << " of bandwidth";
+    ss << nInBoundBlocks() << " inbound and " << nOutBoundBlocks() << " outbound thin blocks have saved "
```

Comment:
Nit: Maybe `block(s)` instead of `block` to properly deal with the singular case :)

Reply:
Indeed!